game.libretro.bsnes-mercury-performance/addon.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="game.libretro.bsnes-mercury" name="SNES (bSNES Mercury)" version="0.94.0" provider-name="Libretro">
+<addon id="game.libretro.bsnes-mercury-performance" name="SNES (bSNES Mercury Performance)" version="0.94.0" provider-name="Libretro">
```

Comment:
How about the name `SNES (bSNES Mercury "Performance")` to show that "Performance" is the name of a profile instead of the emulator?